database.go

```diff
@@ -12,9 +12,9 @@ func loadDatabase(dbFile string) error {
 	}
 	statement := `
 		CREATE TABLE IF NOT EXISTS comments (
-			url text not null,
-			name text not null,
-			comment text not null,
+			url varchar(2083) not null,
+			name varchar(200) not null,
+			comment varchar(10000) not null,
```

Comment:
dont know if those sizes are good. the one I'm sure of is URL, which is set to the max possible url size.